src/storage/src/hummock/shared_buffer/mod.rs

```diff
@@ -91,13 +91,13 @@ pub(crate) type KeyIndexedUncommittedData = BTreeMap<(Vec<u8>, OrderIndex), Unco
 /// the same order index, which means their keys don't overlap.
 pub(crate) type OrderSortedUncommittedData = Vec<Vec<UncommittedData>>;
 
-pub fn to_order_sorted(key_indexed_data: &KeyIndexedUncommittedData) -> OrderSortedUncommittedData {
+pub fn to_order_sorted(key_indexed_data: KeyIndexedUncommittedData) -> OrderSortedUncommittedData {
```

Comment:
Cloning a BTreeMap is more expensive that cloning all its entries so I think we can keep the original implementation.